pgvectorscale/src/access_method/sbq_node.rs

```diff
@@ -1,21 +1,27 @@
 use crate::access_method::node::{ReadableNode, WriteableNode};
 use crate::access_method::PgRelation;
+use crate::util::tape::Tape;
 use crate::util::{ArchivedItemPointer, HeapPointer, ItemPointer, ReadableBuffer, WritableBuffer};
-use pgrx::pg_sys::{InvalidBlockNumber, InvalidOffsetNumber, BLCKSZ};
+use pgrx::pg_sys::{InvalidBlockNumber, InvalidOffsetNumber};
 use pgvectorscale_derive::{Readable, Writeable};
 use rkyv::{vec::ArchivedVec, Archive, Deserialize, Serialize};
 use std::pin::Pin;
 
+use super::stats::{StatsNodeModify, StatsNodeRead, StatsNodeWrite};
+use super::storage::NodeVacuum;
 use super::{
-    meta_page::MetaPage,
-    neighbor_with_distance::NeighborWithDistance,
-    sbq::{SbqQuantizer, SbqVectorElement},
+    meta_page::MetaPage, neighbor_with_distance::NeighborWithDistance, sbq::SbqVectorElement,
     storage::ArchivedData,
 };
 
+/// A node in the SBQ index.  Currently just plain nodes, soon to feature labeled ones.
+pub enum SbqNode {
+    Plain(ClassicSbqNode),
```

Comment:
```suggestion
    Classic(ClassicSbqNode),
```

I think we should be consistent